### DISCUSSION TITLE
x and (e & b) guides not appearing in org mode

### DISCUSSION THREAD

**Discussion by MilayaAvalosAguilar** [meow-edit/meow#612] [Q&A]
Also is there a way to make them not appear for x (as I have visual line number mode) and not ever disappearing for e & b

**Comment by eshrh** [comment]
If I'm understanding your question right, you want the number hints:
- #110 
- https://github.com/meow-edit/meow/blob/master/FAQ.org#why-dont-the-number-hints-for-extending-selection-show-up-in-org-mode-buffers
- https://github.com/meow-edit/meow/blob/master/FAQ.org#why-dont-the-number-hints-for-extending-selection-show-up-in-org-mode-buffers

I think you want:
`(setq meow-expand-exclude-mode-list nil)`

There isn't a way to make e & b use hints and x not without doing some hacking. If you want the hints to never disappear, I would set `meow-expand-hint-remove-delay` to a very large number.
